lib/improver/utilities/spatial.py

```diff
@@ -85,7 +85,7 @@ def check_if_grid_is_equal_area(cube):
 
 
 def convert_distance_into_number_of_grid_cells(
-        cube, distance, max_distance_in_grid_cells):
+        cube, distance, max_distance_in_grid_cells=10**4):
```

Comment:
As `max_distance_in_grid_cells` has been converted into a keyword argument, we'll need to update where this function is called to specify the keyword. I think this just means updating the calls to `convert_distance_into_number_of_grid_cells` in `circular_kernel.py` and `square_kernel.py`, as well as updating the unit tests for `convert_distance_into_number_of_grid_cells`.

Reply:
I was avoiding this as it might be a very large number of changes, but if it's limited will do a grep and update.  Will address the other changes first to keep this separate.